cpp/src/io/parquet/page_data.cu

```diff
@@ -815,6 +815,44 @@ inline __device__ void gpuOutputDecimalAsFloat(volatile page_state_s *s,
   *dst  = (scale < 0) ? (d * kPow10[min(-scale, 39)]) : (d / kPow10[min(scale, 39)]);
 }
 
+/**
+ * @brief Output a fixed-length byte array(len <= 8) as a 64-bit int
+ *
+ * @param[in,out] s Page state input/output
+ * @param[in] src_pos Source position
+ * @param[in] dst Pointer to row output data
+ */
+inline __device__ void gpuOutputFixedLenByteArrayAsInt64(volatile page_state_s *s,
```

Comment:
Does this really need to be volatile?

Reply:
I'm not sure.I added volatile here because all other device decoding function regard `page_state_s` as a volatile variable.

Reply:
Fair enough. :)